covalent/_workflow/lattice.py

```diff
@@ -237,12 +239,22 @@ def build_graph(self, *args, **kwargs) -> None:
             with active_lattice_manager.claim(self):
                 try:
                     workflow_function(*new_args, **new_kwargs)
+                    retval = workflow_function(*new_args, **new_kwargs)
                 except Exception:
                     warnings.warn(
                         "Please make sure you are not manipulating an object inside the lattice."
                     )
                     raise
 
+        from .postprocessing import Postprocessor
+
+        pp = Postprocessor(lattice=self)
+
```

Comment:
Can you put this inside the if condition if it is only valid when the condition is true?

Reply:
Also, if this import is not turning out to be cyclical then maybe we can put it on the top of the file.